components/SubHeader/index.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <div class="subheader">
+    <h1 class="subheader__title">
+      {{ title }}
+    </h1>
+    <p v-if="true" class="subheader__descriptions">
+      {{ descriptions }}
+    </p>
```

Comment:
I think it's should be `v-if="description"` @adzharamrullah 😆

Reply:
Wow my bad, i have fixed that